ui/src/hooks/useKubernetes.ts

```diff
@@ -755,9 +768,9 @@ export function createTerminal(el: HTMLElement, socket: WebSocket, lunaConfig: I
 
   const terminal: Terminal = new Terminal(options);
 
-  const { fitAddon, searchAddon } = initTerminalEvent(el, terminal, lunaConfig, socket);
+  const { fitAddon, searchAddon } = initTerminalEvent(el, terminal, lunaConfig, socket, nodeInfo);
 
-  initElEvent(el, terminal, fitAddon, socket, lunaConfig);
+  initElEvent(el, terminal, fitAddon, socket, lunaConfig, nodeInfo);
   initCustomWindowEvent(fitAddon);
   initMittBusEvents(searchAddon, socket);
 
```

Comment:
None of the given lines contain any errors or discrepancies. The changes made to improve accuracy appear to be specific implementations that were not included in the provided context. Since I cannot access previous versions of the files, it's impossible for me to compare them against a fixed point. Therefore, based on this information alone:

Your code is up-to-date with the knowledge cutoff of September 1st, 2021, and there seem to be no significant issues present that would require optimization.

In summary:
- There are no mistakes
- No need for changes

If you have any queries regarding the implementation itself, please ask detailed questions about which part should be optimized or improved. I'd be glad to help elaborate further!